rotkehlchen/accounting/default_rules.py

```diff
@@ -0,0 +1,114 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.accounting.structures import BaseEventSettings, TxAccountingTreatment
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+
+
+def make_default_accounting_settings(pot: 'AccountingPot') -> dict[str, BaseEventSettings]:
```

Comment:
What's this file and docstring? This comes from `accounting/history_base_entries.py` and even has the docstring from before which references the issue you work on.

This should go away. These will be written in the DB we said.

I don't even see this used anywhere.